src/v2/Apps/Fairs/Components/FairsPromoCarousel.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Children, useEffect, useState } from "react"
+import { Carousel, Clickable, CarouselRail, CarouselCell } from "@artsy/palette"
+import { CarouselProps } from "v2/Components/Carousel"
+
+interface FairsPromoCarouselProps {
+  children: CarouselProps["children"]
+  duration?: number
+}
+
+export const FairsPromoCarousel: React.FC<FairsPromoCarouselProps> = ({
+  children,
+  duration = 5000,
+}) => {
+  const [interacted, setInteracted] = useState(false)
+  const [cursor, setCursor] = useState(0)
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      if (interacted) return
+      setCursor(prevIndex => prevIndex + 1)
+    }, duration)
+
+    return () => {
+      clearInterval(interval)
+    }
+  }, [duration, interacted])
+
+  const initialIndex = cursor % Children.count(children)
+
+  return (
+    <Carousel
+      initialIndex={initialIndex}
+      Next={({
+        children: _children,
+        disabled: _disabled,
+        onClick,
+        ...rest
+      }) => (
+        <Clickable
+          position="absolute"
+          top={0}
+          zIndex={1}
+          width="100%"
+          height="100%"
+          cursor="e-resize"
+          onClick={event => {
+            setInteracted(true)
+            onClick(event)
+          }}
+          {...rest}
+        />
+      )}
+      Previous={() => <></>}
```

Comment:
?

Reply:
Basically just nulls the previous arrow. This is a very weird carousel of course, and it just has a big next button overlaying the whole thing.

Reply:
TBH, we'll probably just drop all this stuff once we have new templates to work with; though it was an interesting exercise to try to recreate it with the new carousel.